interface/src/app/tribe_client/geneset-search-modal.tpl.html

```diff
@@ -0,0 +1,15 @@
+<div class="modal-header">
+    <geneset-search-form organism="organism"><geneset-search-form>
```

Comment:
Only we want this not to be in a modal, but a part of the view. That change will also simplify managing the scope (which is tricky to get right in modals).